Sources/MachOReaderLib/Models/Reports/DyldChainedFixupsReport.swift

```diff
@@ -0,0 +1,187 @@
+import Foundation
+
+// TODO: remove prints
+public struct DyldChainedFixupsReport {
+
+    // MARK: - Properties
+
+    private let file: MachOFile
+    /// The pointer to __LINKED it segment where the LC_DYLD_CHAINED_FIXUPS payload lives.
+    private let baseData: Data
+
+    // MARK: - Lifecycle
+
+    init(file: MachOFile) {
```

Comment:
This init should throw. We can create a common error type for reports and do `ReportError.missingCommand`